CSS/CommonStyles/card.css

```diff
@@ -1,14 +1,15 @@
 .card {
   width: 19rem;
-  height: 30rem;
-  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.7);
+  height: max-content;
   margin: 1rem;
+  background-color: var(--off-white);
 }
 
 .card-img {
-  height: 19rem;
   object-fit: contain;
```

Comment:
Why not "object-fit: cover". It will cover all the space in a container whatever the container's dimension.

Reply:
Fixed it